.golangci.yml

```diff
@@ -0,0 +1,26 @@
+linters:
+  enable-all: true
+  disable:
+    - gci
+    - gofmt
+    - gofumpt
+    - goimports
+    - golint
+
+issues:
+  exclude:
+    - use of internal package
+    - parameter name 'w|r' is too short for the scope of its usage
```

Comment:
`Handler(w http.ResponseWriter, r *http.Request)`
Short names for variables but it's a idiomatic names for http.HandlerFunc

Reply:
You probably should just disable the whole linter.